### PR TITLE
Create Remove-TeamTargetingHierarchy.md

### PR DESCRIPTION
**Pull request by ayrtonsoftware**

### CODE REVIEW

File: teams/teams-ps/teams/Remove-TeamTargetingHierarchy.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Removes the tenant's hierarchy.
+
+## SYNTAX
+
+### Remove (Default)
+```
+Remove-TeamTargetingHierarchy
+```
+
+## DESCRIPTION
```
Comment (tseward) [inline teams/teams-ps/teams/Remove-TeamTargetingHierarchy.md:23]: Can you add a description, even if it is identical to the Synopsis?

**Review by tseward — CHANGES_REQUESTED** [review]

File: teams/teams-ps/teams/Remove-TeamTargetingHierarchy.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
```
Comment (get-itips) [inline teams/teams-ps/teams/Remove-TeamTargetingHierarchy.md:6]: this should be a github username

File: teams/teams-ps/teams/Remove-TeamTargetingHierarchy.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## DESCRIPTION
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-TeamTargetingHierarchy
+```
+
```
Comment (get-itips) [inline teams/teams-ps/teams/Remove-TeamTargetingHierarchy.md:31]: Here should be an example's description

**Review by dariomws — APPROVED** [review]
I'm ok after the suggestion. Thanks!

**Comment by yogkumgit** [comment]
@ayrtonsoftware

**Comment by yogkumgit** [comment]
Gentle reminder @ayrtonsoftware Please implement the suggestions. Thanks

**Comment by yogkumgit** [comment]
@Kateyanne Please help

**Comment by Kateyanne** [comment]
@get-itips are you able to implement the suggestions yourself?

**Comment by get-itips** [comment]
> @get-itips are you able to implement the suggestions yourself?

I am afraid only someone with push permissions on the repo or the PR author

**Review by tseward — APPROVED** [review]

**Pull request merged**